requirements/edx/base.txt

```diff
@@ -56,7 +56,6 @@ edx-organizations==0.4.5
 edx-rest-api-client==1.7.1
 edx-search==1.1.0
 edx-submissions==2.0.9
-edxval==0.1.1
```

Comment:
Note: This is just for sandbox and for jenkins tests. Add this back this before merging PR.